### ISSUE TITLE
Readme problem with making new SSH keys

### ISSUE THREAD

**Issue by ChantalJuntao** [HolyLab/HolyLabRegistry#196]
Making a new SSH key using the ReadMe produces this error
```
(@v1.6) pkg> registry add git@github.com:HolyLab/HolyLabRegistry.git
     Cloning registry from "git@github.com:HolyLab/HolyLabRegistry.git"
ERROR: failed to clone from git@github.com:HolyLab/HolyLabRegistry.git, error: GitError(Code:EEOF, Class:SSH, ERROR: You're using an RSA key with SHA-1, which is no longer allowed. Please use a newer client or a different key type.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.

)
```
It seems like github has dropped support for certain key types. Do we need to update the readme?

**Comment by ChantalJuntao** [comment]
Also, I'm pretty sure 
```
$ ssh-keygen -m PEM rsa -b 4096 -C "your_email@example.com"
```
should be 
```
$ ssh-keygen -m PEM b 4096 -C "your_email@example.com"
```

**Comment by ChantalJuntao** [comment]
Update neither of the previous keygen methods work anymore since Github now requires authentification with sha-2. I used 
```
$ ssh-keygen -t ecdsa -b 521 -C "your_email@example.com"
$ eval "$(ssh-agent -s)"
$ ssh-add ~/.ssh/id_ecdsa
```
I'll submit a pull request updating the readme, although I'm not sure what still needs to be changed for keys on windows.

**Comment by ChantalJuntao** [comment]
Pull Request #197

**Comment by timholy** [comment]
Closed by #197
